app/views/candidate_interface/adviser_sign_ups/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title, t('page_titles.new_adviser_sign_up') %>
-<% content_for :before_content, govuk_back_link_to(new_candidate_interface_adviser_sign_up_path(preferred_teaching_subject_id: @adviser_sign_up_form.preferred_teaching_subject_id), t('.back')) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_details_path, t('.back')) %>
```

Comment:
I think I would expect this to go back to the last place, the interruption page, rather than the candidate details. Might be worth checking with Pete